plugins/pagerduty/src/api/client.ts

```diff
@@ -80,11 +92,11 @@ export class PagerDutyClient implements PagerDutyApi {
   }
 
   triggerAlarm({
-    integrationKey,
-    source,
-    description,
-    userName,
-  }: TriggerAlarmRequest): Promise<Response> {
+                 integrationKey,
```

Comment:
Prettier is currently complaining about these lines.

Reply:
fixed